smt/utils/kriging_utils.py

```diff
@@ -20,11 +20,13 @@
 # TODO: Create hyperclass Kernels and a class for each kernel
 
 
-def standardization(X, y, copy=False):
+def standardization(X, y, copy=False, scale_X_to_unit=False):
```

Comment:
I've just noticed we never set copy or scale_X_to_unit options to True, so we just have to remove them.

Reply:
I agree with copy (not sure when it will True) but scale_X_to_unit may interact with future developments. I would prefer to keep it.